_posts/2019-07-12-barcelona-announcement.md

```diff
@@ -2,29 +2,29 @@
 
 ![Barcelona](/assets/posts/barcelona-city.jpg)
 
-As a touristic destination, Barcelona is known for its beautiful architecture like Sagrada Familia, Casa Batlló, Ciutadella’s park and Palau de la Musica; for its delicious gastronomy and for its multicultural and open personality.
+As a touristic destination, Barcelona is known for its beautiful architecture like [Sagrada Familia](https://en.wikipedia.org/wiki/Sagrada_Fam%C3%ADlia), [Casa Batlló](https://en.wikipedia.org/wiki/Casa_Batll%C3%B3), [Ciutadella’s park](https://en.wikipedia.org/wiki/Parc_de_la_Ciutadella) and [Palau de la Musica](https://en.wikipedia.org/wiki/Palau_de_la_M%C3%BAsica_Catalana); for its [delicious gastronomy](https://en.wikipedia.org/wiki/Catalan_cuisine) and for its multicultural and open personality.
 
-There are lots of ways to enjoy the city: you can go to the beach, have a hike on some green areas like Park Güell or just take a walk through the different neighborhoods in order to experience how the influence of History has affected the city.
+There are lots of ways to enjoy the city: you can go to the beach, have a hike on some green areas like [Park Güell](https://en.wikipedia.org/wiki/Park_G%C3%BCell) or just take a walk through the different neighborhoods in order to experience how the influence of [History](https://en.wikipedia.org/wiki/History_of_Barcelona) has affected the city.
 
 ## Save the date!
 
 We have finally a date for this year's edition!
 
 Rustfest 2019 will start on **November 9th** (Saturday) and will end on **November 12th** (Tuesday).
 
-As usual, this edition will have two first days full of talks and workshops and two additional days (*aka Impl Days*) where you will be able to get together with other rustaceans, meet in smaller groups and work on your own community projects.
+As usual, this edition will have two first days full of talks and workshops and two additional days (*aka Impl Days*), where you will be able to get together with other rustaceans, meet in smaller groups and work on your own community projects.
 
 ## Mixing things a little bit
 
-Although we are keeping the same length in terms of days we have decided to change things a little bit and try something different.
+Although we are keeping the same length in terms of days, we have decided to change things a little bit and try something different.
 
-This year, the conference format will change and instead of a first day of talks and a second day of workshops we're going to try to split the days into talks in the morning and workshops in the afternoon.
+This year, the conference format will change, and instead of a first day full of talks and a second day of workshops, we'll try to split the days in two parts, the talks in the morning and workshops in the afternoon.
```

Comment:
```suggestion
This year, the conference format will change, and instead of a first day full of talks and a second day full of workshops, we'll try to split the days in two different parts, with talks in the morning and workshops in the afternoon.
```